ocs_ci/utility/ibm_hci.py

```
@@ -0,0 +1,721 @@
+        """
+        log.info(
+            f"Performing {operation} operation on node {node_name} (force={force}, wait={wait})"
+        )
+
+        # Detach BareMetalHost before power off/cycle/reset to prevent auto-recovery
+        if operation in ["off", "cycle", "reset"]:
+            log.info("Detaching BareMetalHost to prevent auto-recovery")
+            self._detach_baremetalhost(node_name)
+
+        # Get node details
+        rack_serial, node_info, rack_ip = self._get_node_details_by_name(node_name)
```

Ruff (0.15.12)

[warning] Unpacked variable `rack_serial` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)